### DISCUSSION TITLE
Daily 26/9

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#32] [General]
El viernes termine con la lectura de los apuntes de CSS y además hice un poco de práctica. Participé de la clase sincrónica de Inglés y luego segui con el repaso de los contenidos para rendir el exámen.
El  fin de semana participe de la reunión de  Proyecto. Completé la seccion de Resumen del ERS. Retomé el estudio de Python en la plataforma de Cisco. También rendí el exámen de Inglés.
Para hoy cuento con poca disponibilidad. Voy a agregar US de martillero y continuar con Cisco.

**Comment by AhumadaCeleste** [comment]
El viernes vi clases atarazadas. Continuo practicando html. Hoy seguiré en vistas del venidero examen. Mañana quisiera iniciar con css

**Comment by Brian-Almada** [comment]
El viernes no pude tocar nada ya que estuve fuera de casa todo el día.
Hoy continuó con el estudio de Cisco e ingles.

**Comment by rpgrca** [comment]
Hice algunos de los ejercicios propuestos de Python para repasar Cisco.

**Comment by IngridAlcocer05** [comment]
Hoy continue con Python y realice el parcial de ingles.
